includes/Modules/Sign_In_With_Google.php

```diff
@@ -107,4 +109,68 @@ protected function setup_assets() {
 	protected function setup_settings() {
 		return new Settings( $this->options );
 	}
-}
+
+	/**
+	 * Registers and Enqueues the sign in button script.
+	 *
+	 * @since n.e.x.t
+	 */
+	protected function register_and_enqueue_script() {
+		$sign_in_script = new Script(
+			'googlesitekit-sign-in-with-google-sign-in-button',
+			array(
+				'src'       => 'https://accounts.google.com/gsi/client',
+				'execution' => 'async',
+			)
+		);
+		$sign_in_script->register( $this->context );
+		$sign_in_script->enqueue();
+	}
```

Comment:
Let's add this script in the inline script that will inject markup for the button.